heft-plugins/heft-storybook-plugin/src/StorybookPlugin.ts

```diff
@@ -139,8 +163,8 @@ export class StorybookPlugin implements IHeftPlugin<IStorybookPluginOptions> {
         bundle.hooks.configureWebpack.tap(
           { name: PLUGIN_NAME, stage: Number.MAX_SAFE_INTEGER },
           (webpackConfiguration: unknown) => {
-            // Discard Webpack's configuration to prevent Webpack from running
-            return null;
+            // Discard Webpack's configuration to prevent Webpack from running only when starting a storybook server
+            return storybookStaticBuildParameters.actionAssociated ? webpackConfiguration : null;
```

Comment:
Alternately, I would return `null` if `storybookParameters.actionAssociated && storybookParameters.value` is true, since we would only want to take over from the normal bundler if in start mode AND `--storybook` is specified.

Reply:
Updated, thanks!